packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -451,8 +451,11 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
 }
 
 /// A [FlutterGoldenFileComparator] for skipping golden image tests when the
-/// current environment is not supported. This comparator is used in post-submit
-/// checks on LUCI.
+/// current environment is not supported.
```

Comment:
This doc comment is confusing. It's not clear to me when the FlutterSkippingGoldenFileComparator or when the FlutterLocalFileComparator is used. The first sentence makes it sound like the Skipping is the default catch-all if nothing else works, the second sentence seems to claim its the Local one.